README.md

```diff
@@ -8,6 +8,19 @@ Need help with Node.js? You've come to the right place!
 If you have a question about Node.js, [create a new issue](https://github.com/nodejs/help/issues/new)
 and ask there!
 
+When inserting you're code:
+use 3 backticks, right under the tilde & atop of the tab key followed by js: then end your code with 3 tilde. As seen below, except without the round brackets.  
```

Comment:
And I was for a short while confused by the 'right under the tilde & atop of the tab key' bit. Maybe it could be clarified that it is about keyboard layout. May be a scrennshot like this, but not that ugly?

![bt](https://cloud.githubusercontent.com/assets/10393198/17458808/5b7352a4-5c28-11e6-9bdf-0d38bc07ae60.png)